test/status/recipes/svn.yaml

```diff
@@ -0,0 +1,15 @@
+checkoutSCM:
+    - 
+      scm: svn
+      url: svn://svn.code.sf.net/p/pc1d/code/trunk
```

Comment:
The `svnserve` command should be available. The test case could spawn a server (after creating a repo) and the test could import some files. At the end the test must reap the server though.

```sh
svnadmin create foo
cat >foo/conf/svnserve.conf <<EOF
[general]
anon-access = write
EOF
svnserve -d --foreground -r .
```